<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>183</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Map</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>141</width>
         <height>62</height>
        </rect>
       </property>
       <property name="title">
        <string>Nodes</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="nodeComboBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>141</width>
         <height>62</height>
        </rect>
       </property>
       <property name="title">
        <string>Links</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="linkComboBox"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Time</string>
       </property>
       <widget class="QPushButton" name="nextButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>28</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="timeLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>35</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="previousButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>28</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>View</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>151</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>name to search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
